clang/lib/Sema/BoundsAnalysis.cpp

```diff
@@ -752,10 +750,34 @@ BoundsMapTy BoundsAnalysis::GetWidenedBounds(const CFGBlock *B) {
   return EB->In;
 }
 
+Expr *BoundsAnalysis::GetTerminatorCondition(const Expr *E) const {
+
+  if (!dyn_cast<BinaryOperator>(E->IgnoreParens())){
+    if (auto *CE = dyn_cast<CastExpr>(E))
+      if (CE->getCastKind() == CastKind::CK_IntegralCast)
+        E = CE->getSubExpr();
+
+    return const_cast<Expr *>(E);
+ }
+  const auto *BO = dyn_cast<BinaryOperator>(E->IgnoreParens());
+  E = GetTerminatorCondition(BO->getRHS()->IgnoreParens());
+	// According to C11 standard section 6.5.13, the logical AND Operator
```

Comment:
It seems there are tabs preceding each line of the comment here. Please replace them with spaces.